src/server/dfly_main.cc

```diff
@@ -381,9 +384,12 @@ bool RunEngine(ProactorPool* pool, AcceptServer* acceptor) {
       delete admin_listener;
     } else {
       LOG(INFO) << "Listening on " << printable_addr;
+      listeners.push_back(admin_listener);
```

Comment:
I would rather not expose admin connections via "client list".

Reply:
We don't really have a well defined security boundary between and non-admin connections yet, and I'd rather not invent it on the fly... for example, should admin connections be hidden even when CLIENT LIST is executed from the admin connection?

Reply:
Ok, opned https://github.com/dragonflydb/dragonfly/issues/1495 for that